macros/recursive_dag.sql

```diff
@@ -15,35 +15,8 @@ with recursive direct_relationships as (
 
 -- recursive CTE
 -- one record for every resource and each of its downstream children (including itself)
-all_relationships (
-    parent_id,
-    parent,
-    parent_resource_type,
-    parent_model_type,
-    parent_materialized,
-    parent_access,
-    parent_is_public,
-    parent_source_name,
-    parent_file_path,
-    parent_directory_path,
-    parent_file_name,
-    parent_is_excluded,
-    child_id,
-    child,
-    child_resource_type,
-    child_model_type,
-    child_materialized,
-    child_access,
-    child_is_public,
-    child_source_name,
-    child_file_path,
-    child_directory_path,
-    child_file_name,
-    child_is_excluded,
-    distance,
-    path,
-    is_dependent_on_chain_of_views
-) as (
+all_relationships {{ recursive_dag__all_relationships_cte_aliases() }}
```

Comment:
I've removed changes to this macro completely and added a couple aliases to make `bigquery__recursive_dag` work, since that was already required to support clickhouse 24.3 lte